components/alert/AlertResult.vue

```diff
@@ -113,6 +130,29 @@ export default Vue.extend({
         ...FactionBgClass(this.alert.result.victor),
       }
     },
+    roundByPhase(): string {
+      if (this.alert.ps2alertsEventType !== Ps2alertsEventType.OUTFIT_WARS_AUG_2022) {
+        return ''
+      }
+      const match = this.alert as InstanceOutfitWarsResponseInterface;
+      if (match.phase === Phase.QUALIFIERS) {
+        return match.round.toString()
+      } else if (match.phase === Phase.PLAYOFFS) {
```

Comment:
I'm a bit confused by this - is it spitting out the round number as a string here, or is there other magic going on that translates it to say "Playoffs" or "Qualifiers"?

Reply:
Its spitting out the round number as a string - I made a new filter for that to make it more clear